cmd/insprctl/cli/init.go

```diff
@@ -34,11 +35,13 @@ var initCommand = cmd.NewCmd("init").
 	).NoArgs(
 	func(c context.Context) error {
 		config := insprConfiguration{}
-		fmt.Print("enter insprd host (http://localhost:8080):")
+		fmt.Print("enter insprd IP (http://localhost:8080):")
 		fmt.Scanln(&config.ServerIP)
 		if config.ServerIP == "" {
 			config.ServerIP = "http://localhost:8080"
 		}
+		fmt.Print("enter insprd host (http://example.inspr.dev) Opitional config:")
```

Comment:
I would change it to something like
`Optional: insprd host (example.inspr.dev):`

But is best to check if @luiz-couto and @LucasStarlingdePaulaSalles